### PR TITLE
Use TYPO3 v8.7

### PR DESCRIPTION
**Pull request by haogatyp**

### CODE REVIEW

**Comment by haogatyp** [comment]
I changed the typo3 dependency from 7.6 to 8.7

File: Dockerfile [hunk 1]
```diff
@@ -46,11 +46,12 @@ ADD . /app
 WORKDIR /var/www/html
 RUN composer config repositories.t3ter composer https://composer.typo3.org && \
     composer config repositories.kitodo-publication path /app && \
+		composer config repositories.kitodo-presentation path /app && \
```
Comment (claussni) [inline Dockerfile]: This is an error. Kitodo.Presentation is not in /app/

File: Dockerfile [hunk 2]
```diff
@@ -46,11 +46,12 @@ ADD . /app
...
     composer config minimum-stability dev && \
     composer config prefer-stable true
 RUN composer require sjbr/static-info-tables:6.5.1 && \
     composer require devlog/devlog:~3.0.4 && \
-    composer require typo3-ter/dlf:~2.2.0 && \
+    composer require kitodo/presentation  && \
```
Comment (claussni) [inline Dockerfile]: Is this always the latest version of Kitodo.Presentation?

**Comment by claussni** [comment]
Branch is messed up through GitHub WebEditor merge. I cherry picked the important commits into master.

**Pull request closed without merging**